app/src/components/organisms/Appointment.tsx

```diff
@@ -2,33 +2,29 @@ import React from 'react';
 import Query from 'devextreme/data/query';
 import { formatDate } from 'devextreme/localization';
 import classData from 'data/clean.json';
-import { ICalendarItem } from 'components/shared/interfaces/timetable.interfaces';
+import { ICalendarTeacherItem } from 'components/shared/interfaces/timetable.interfaces';
 import { Grid } from '@mui/material';
-import { parseCalendarJSON } from 'utils/utils';
+import { parseCalendarTeacher } from 'utils/utils';
 import { Container } from '@mui/system';
 
-const data: ICalendarItem[] = parseCalendarJSON(JSON.parse(JSON.stringify(classData)));
+const data: ICalendarTeacherItem[] = parseCalendarTeacher(JSON.parse(JSON.stringify(classData)));
 
-/**
- * Filter newdata by course id
- */
-
-export function getClassById(id: string) {
-  return Query(data).filter(['courseId', id]).toArray()[0];
+function getTeacherById(id: number) {

```

Comment:
There's a current error on the tests not passing, it's because this line needs to be exported for the Appointment.test.tsx test